lib/webtask.js

```diff
@@ -708,6 +756,66 @@ exports.ensure_authenticated = function(ctx) {
             });
         }
 
+        function validate_auth_token(auth) {
+            var auth_token;
+            var decoded_auth_token;
+            var userinfo;
+            var aud_should_be;
+            var secret = auth.secret || ctx.config.default_auth0_client_secret.current;
```

Comment:
Given the scope of changes we discussed on Slack yesterday, webtasks should be completely unaware of Auth0 existence. Why do we need the default_auth0_client_secret here? @woloski is this required for any part of the UX you had in mind?